Dockerfile

```diff
@@ -34,7 +35,8 @@ ENV CRIU_VERSION 1.7
 RUN mkdir -p /usr/src/criu \
     && curl -sSL https://github.com/xemul/criu/archive/v${CRIU_VERSION}.tar.gz | tar -v -C /usr/src/criu/ -xz --strip-components=1 \
     && cd /usr/src/criu \
-    && make install-criu
+    && make install-criu \
+    && make clean
```

Comment:
Silly question, why don't we just do an `rm -rf /usr/src/criu`? Once it's installed do we need the source code lying  around?